pkg/controllers/managedzone/managedzone_controller.go

```diff
@@ -164,7 +165,11 @@ func (r *ManagedZoneReconciler) SetupWithManager(mgr ctrl.Manager) error {
 
 func (r *ManagedZoneReconciler) publishManagedZone(ctx context.Context, managedZone *v1alpha1.ManagedZone) error {
 
-	mzResp, err := r.DNSProvider.EnsureManagedZone(managedZone)
+	DNSProvider, err := r.DNSProvider(ctx, managedZone)
```

Comment:
DNSProvider -> dnsProvider

Reply:
Sorry i made those changes but i must not have committed it